src/components/dropdown/README.md

```diff
@@ -1 +1,138 @@
+</div>
+```
+
+### Using a heading within the menu list
+
+To add a heading within the menu list, add the following heading markup:
+
+```
+<div class="rvt-dropdown__menu-heading" aria-hidden="true">Menu list heading</div>
+```
+
+Within the menu list markup:
+
+```
+<div class="rvt-dropdown__menu" id="dropdownHeading" role="menu" hidden data-dropdown-menu>
+  <a href="#" role="menuitemradio">Item One</a>
+  <a href="#" aria-checked="true" role="menuitemradio">Item Two</a>
+  <a href="#" role="menuitemradio">Item Three</a>
+  <div class="rvt-dropdown__menu-heading" aria-hidden="true">Menu list heading</div>
+    <div role="group" aria-label="Related">
+        <a href="#">Nested item one</a>
+        <a href="#">Nested item two</a>
+    </div>
+  </div>
+</div>
+```
+
+## Initialization
+
+In order to initialize the dropdown as an ES6 module, you will need to create a new instance of it, and pass it the element you wish to instantiate as a dropdown.
+
+```
+const dropdownElement = document.querySelector('[data-dropdown="your-data-dropdown-value"]');
+document.newDropdown = new Rivet.Dropdown(dropdownElement);
+```
+
+## "Public" methods
+
+If you use the appropriate data attribute/id combination in your markup, the dropdown will work without the need for any additional JavaScript. But if you need to control the dropdown programmatically, there are several methods from the Rivet dropdown's API you can use:
+
+| Method           | Description                                                                            |
+| ---------------- | -------------------------------------------------------------------------------------- |
+| .open()  | Opens the dropdown menu list |
+| .close() | Closes the dropdown menu list |
+| .init()  | Adds the built-in event listeners to the dropdown |
+| .destroy() | Removes all built-in event listeners from the dropdown |
+
+## Custom events
+
+| Event        | Description                                                                                                                                                                                                                                                          |
+| ------------ | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `dropdownOpen`  | Emitted when the dropdown is opened (using the Dropdown.open() method). The value of the dropdown id attribute is also passed along via the custom event’s detail property and is available to use in your scripts as event.detail.id()  |
+| `dropdownClose` | Emitted when the dropdown is closed (using the Dropdown.close() method). The value of the dropdown id attribute is also passed along via the custom event’s detail property and is available to use in your scripts as event.detail.id() |
```

Comment:
See same comment in #359 about the `event.detail` function vs. regular object property. Should be able to document as `event.detail.id` without parens.

```suggestion
| `dropdownClose` | Emitted when the dropdown is closed (using the Dropdown.close() method). The value of the dropdown id attribute is also passed along via the custom event’s detail property and is available to use in your scripts as event.detail.id |
```